vipps-ecom-api.md

```diff
@@ -1437,21 +1437,21 @@ is always to query the eCom API once you arrive back in the merchant app/website
 
 The sections below explain in more detail how to integrate for browsers and apps.
 
-## App-switch between browser and the Vipps app
```

Comment:
This seemed redundant, as its mentioned under https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#desktop-browser-initiated-payments and